sdk/examples/xo_python/sawtooth_xo/processor/handler.py

```diff
@@ -50,47 +48,57 @@ def apply(self, transaction, context):
 
         xo_payload = XoPayload.from_bytes(transaction.payload)
 
-        # 2. Retrieve the game data from state storage
-        board, state, player1, player2, game_list = \
-            _get_state_data(context, self._namespace_prefix, xo_payload.name)
+        xo_context = xo_state.XoState(context, xo_payload.name)
```

Comment:
Let's call this `xo_state` to match the class

Reply:
Sounds good.